pages/blog/node-js-and-ipv6.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Binding a Node.js Web app to IPv4 and IPv6 Sockets"
+date: "2021-08-05"
+author: "Evan Tahler"
+pullQuote: "How does Grouparoo use Next.js to load pages and components from plugins to modify our web user interface?"
```

Comment:
Is this pullQuote from another blog post?

Reply:
Whoops! Thanks!

```suggestion
pullQuote: "Grouparoo can speak IPv4 and IPv6 - Here's how we did it."
```